app/src/main/res/values/dimens.xml

```diff
@@ -377,7 +377,7 @@
   <dimen name="general_button_item_split_view_margin_top_left">20dp</dimen>
   <dimen name="general_button_item_split_view_margin_top_right">68dp</dimen>
   <dimen name="general_button_item_margin_top">20dp</dimen>
-  <dimen name="continue_submit_button_width">88dp</dimen>
+  <dimen name="continue_submit_button_width">180dp</dimen>
```

Comment:
@Ryggs You can remove this value from all `dimens` now as it might not be getting used.